config/locales/crowdin/af.yml

```diff
@@ -269,7 +269,7 @@ af:
       notice_reset_token: "A new %{type} token has been generated. Your access token is:"
       token_value_warning: "Note: This is the only time you will see this token, make sure to copy it now."
       no_results_title_text: There are currently no access tokens available.
-      notice_ical_tokens_reverted: "iCalendar token \"%{token_name}\" for calendar \"%{calendar_name}\" of project \"%{project_name}\" has been revoked. The iCalendar URL with this token is now invalid."
+      notice_ical_token_revoked: "iCalendar token \"%{token_name}\" for calendar \"%{calendar_name}\" of project \"%{project_name}\" has been revoked. The iCalendar URL with this token is now invalid."
```

Comment:
Doesn't really matter but altering the files in `config/locales/crowdin/` is superfluous. They all are based on the `en.yml` and `js-en.yml` files, transmitted to Crowdin and then pulled from there again after being translated. So the line would be removed by the next pull from Crowdin anyway, as the key is removed from `en.yml`.

Reply:
I needed to adapt some translation files as GitHub raised merge conflicts while merging the dev branch back into this feature branch. I kind of thought as well that the crowdin workflow will make these changes obsolete at the end.